app/services/idv/session.rb

```diff
@@ -5,13 +5,11 @@ class Session
       applicant
       go_back_path
       idv_phone_step_document_capture_session_uuid
-      idv_gpo_document_capture_session_uuid
       vendor_phone_confirmation
       user_phone_confirmation
       pii
       previous_phone_step_params
       previous_profile_step_params
```

Comment:
I see we removed the only remaining reference to this in `gpo_controller.rb`. Should we remove it?

Reply:
yes, good catch!